scripts/deploy_assisted_service.sh

```diff
@@ -17,13 +19,14 @@ export ENABLE_KUBE_API=${ENABLE_KUBE_API:-false}
 export ENABLE_KUBE_API_CMD="ENABLE_KUBE_API=${ENABLE_KUBE_API}"
 export OPENSHIFT_VERSIONS=${OPENSHIFT_VERSIONS:-}
 export OPENSHIFT_VERSIONS_CMD=""
+export DEBUG_SERVICE=${DEBUG_SERVICE:-}
```

Comment:
I think it would be easier to use the same `DEBUG` as for debugging python

Reply:
why? I give the control to the developer to define exactly what do debug

Reply:
I understand that. I just think that when a contributor comes to a repo he would think about `DEBUG` and not specifically what to debug. The same goes for debugging the python that it's `DEBUG` and not `DEBUG_PYTHON`

Reply:
I am not sure that setting one parameter to debug them all it's a good idea...
IMO the service should be always debuggable (add it to my bashrc or somthing) but not the E2E.

Reply:
Could the e2e debugging be running in the background as well? I guess that's the desirable behavior. Then whoever would want to debug, just turns on the `DEBUG` flag and the entities would be debuggable, no strings attached :)

Reply:
It's possible but not desired, the test might failed until you connected.

Reply:
Then the developer should put the breakpoint before running :)
Honestly, I think this debug behavior is preferred

Reply:
no, If you want to debug the beginning of the test you never succeed.
I debugged it so many times and i don't agree but we can give it a try and change it in the future.

Reply:
After discussion, it seems that python behavior can't be aligned to service behavior.
Breakpoints can't be predefined unless attached, and it takes time to attach to a remote.

So we will keep the 2 flags for flexibility and the behavior by design.